certify-service/src/main/java/io/mosip/certify/config/AppConfig.java

```diff
@@ -53,4 +69,31 @@ public RestTemplate restTemplate() {
         return new RestTemplate(requestFactory);
     }
 
+    @Override
+    public void run(ApplicationArguments args) throws Exception {
+        log.info("===================== IDP_SERVICE ROOT KEY CHECK ========================");
+        String objectType = "CSR";
+        KeyPairGenerateRequestDto rootKeyRequest = new KeyPairGenerateRequestDto();
+        rootKeyRequest.setApplicationId(Constants.ROOT_KEY);
+        keymanagerService.generateMasterKey(objectType, rootKeyRequest);
+        log.info("===================== IDP_SERVICE MASTER KEY CHECK ========================");
+        KeyPairGenerateRequestDto masterKeyRequest = new KeyPairGenerateRequestDto();
+        masterKeyRequest.setApplicationId(Constants.OIDC_SERVICE_APP_ID);
+        keymanagerService.generateMasterKey(objectType, masterKeyRequest);
+
+        if(!StringUtils.isEmpty(cacheSecretKeyRefId)) {
+            SymmetricKeyGenerateRequestDto symmetricKeyGenerateRequestDto = new SymmetricKeyGenerateRequestDto();
+            symmetricKeyGenerateRequestDto.setApplicationId(Constants.OIDC_SERVICE_APP_ID);
+            symmetricKeyGenerateRequestDto.setReferenceId(cacheSecretKeyRefId);
+            symmetricKeyGenerateRequestDto.setForce(false);
+            keymanagerService.generateSymmetricKey(symmetricKeyGenerateRequestDto);
+            log.info("============= IDP_SERVICE CACHE SYMMETRIC KEY CHECK COMPLETED =============");
```

Comment:
Change all the references in the logs from IDP_ to CERTIFY_

Reply:
changed